libs/ui-lib/lib/common/utils.ts

```diff
@@ -48,3 +52,7 @@ export const validateFileSize = (value: string, fileSize: number): boolean => {
 export const getMaxFileSizeMessage = (fileSize: number): string => {
   return `File size is too big. Upload a new ${fileSize / 1000} kB or less.`;
```

Comment:
From a presentational standpoint the limit is 1MB, behind the hood we are allowing 2MB because the YAML content gets encoded to Base64 and that bloats the file size by almost 133%. So the UI must mention 1MB but allow 2MB

Reply:
ok. I'll change the message then.